src/modules/Sidebar/Sidebar.js

```diff
@@ -41,6 +42,12 @@ class Sidebar extends Component {
 
     /** Sidebar width. */
     width: PropTypes.oneOf(['very thin', 'thin', 'wide', 'very wide']),
+
+    /** Controls whether or not to close sidebar when click outside */
+    closable: PropTypes.bool,
+
+    /** Function to change "visible" prop */
+    toggleVisibility: PropTypes.func,
```

Comment:
Looking closer at the core API docs for Sidebar callbacks, I think we need to go with these:

![image](https://user-images.githubusercontent.com/5067638/28487449-5c7d5eee-6e45-11e7-93d7-3d06ac4f45b6.png)

https://semantic-ui.com/modules/sidebar#settings